automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -93,8 +94,9 @@ trait GPAllocBeforeAndAfterAll extends BeforeAndAfterAll with BillingFixtures wi
 
 final class LeonardoSuite
     extends Suites(
+      new RuntimeCreationPdSpec,
       new ClusterStatusTransitionsSpec,
-      new LabSpec,
+//      new LabSpec,
```

Comment:
this is what I had to do to make tests passing...I think maybe `Suites` has some upper limit in terms of how many specs we can mix in.